core/sql/optimizer/UdfDllInteraction.cpp

```diff
@@ -1214,6 +1214,29 @@ NABoolean TMUDFInternalSetup::setTypeInfoFromNAType(
           switch (length)
             {
               // SMALLINT, INT, LARGEINT, NUMERIC, signed and unsigned
+            case 1:
+              if (CmpCommon::getDefault(TRAF_TINYINT_SPJ_SUPPORT) == DF_ON)
```

Comment:
Did you mean DF_OFF? Seems odd that if TRAF_TINYINT_SPJ_SUPPORT is 'ON' then we *wouldn't* support tinyint.